Wikis aren't crawled. Suggest GH Pages alternative (#5054)
github/docs: content/communities/documenting-your-project-with-wikis/about-wikis.md

You can edit wikis directly on {% data variables.product.product_name %}, or you can edit wiki files locally. By default, only people with write access to your repository can make changes to wikis, although you can allow everyone on {% data variables.product.product_location %} to contribute to a wiki in {% if currentVersion == "github-ae@latest" %}an internal{% else %}a public{% endif %} repository. For more information, see "[Changing access permissions for wikis](/communities/documenting-your-project-with-wikis/changing-access-permissions-for-wikis)".

{% note %}

> **Note:** Crawlers such as those from search engines will not crawl the contents of wikis as they are excluded by `robots.txt` for anti-abuse reasons. While editing is not as accessible as a wiki, consider using [{% data variables.product.prodname_pages %}](https://pages.github.com/) backed by a public repository for content that can be crawled and accepting of public contributions.

Reviewer: ```suggestion
**Note:** Search engines will not index the contents of wikis. To have your content indexed by search engines, you can use [{% data variables.product.prodname_pages %}](/pages) in a public repository.
```
Thanks so much for adding this information @nelsonjchen! I simplified the language a bit, since our wiki content is frequented by beginners.
